src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,21 @@
+import {Link} from "react-router-dom";
+
+export const ArticleCard = ({article_id, title, topic, author, body, created_at, votes, comment_count}) => {
+    return (
+        
+        <article className='article-card'>
+            <Link to={`/${article_id}`}>
```

Comment:
Having a link to nowhere is not the best UI - I, again, know this behaviour will be in place in the next ticket!